notebooks/official/pipelines/kfp2_pipeline.ipynb

```diff
@@ -0,0 +1,1049 @@
+{
```

Comment:
<p>XgBoost &gt; XGBoost</p><p> </p><p>Deploys &gt; Deploy</p><p> </p><p>The components link leads to 404 Page not found.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>